smack-extensions/src/main/java/org/jivesoftware/smackx/nick/NickListener.java

```diff
@@ -0,0 +1,29 @@
+/**
+ *
+ * Copyright 2018 Miguel Hincapie.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smackx.nick;
+
+import org.jivesoftware.smack.packet.Message;
+
+/**
+ * Used with NickManager.
+ *
+ * @author Miguel Hincapie 2018
```

Comment:
I would prefer if you would not put a data in the @author information.

Reply:
what do you mean? should I remove @autor from the file?

Reply:
Sorry, I meant `date`.

Reply:
done